doc/adrs/replace-migration-scaffold-with-clean-scaffold-in-master-branch.md

```diff
@@ -0,0 +1,34 @@
+---
+title: "ADR: Replace migration Scaffold with clean Scaffold in Master Branch"
+date: 2018-08-16
+draft: false
+status: "Accepted"
+---
+
+# ADR: Replace migration Scaffold with clean Scaffold in Master Branch
+
+## Context
+At the moment, the master branch has the scaffold specifically build for helping developers to migrate v1 apps to v2. From monitor, the majority usage comes from [App Migrator](https://github.com/zendesk/zendesk_app_migrator) and [ZAT](https://github.com/zendesk/zendesk_apps_tools) which use the master branch as the download source.
+It should be more helpful to show a *clean*(from scratch) scaffold to the developers directly come to this repo for a starting point to build apps
+
+## Decision
+
+### New app scaffold
+A new *clean* app scaffold has been build on branch [offapps-migration](https://github.com/zendesk/app_scaffold/tree/offapps-migration) based on branch [from-scratch](https://github.com/zendesk/app_scaffold/tree/from-scratch)
+
+### Move migration scaffold into App Migrator repository
+Migration scaffold in master branch will be moved into [App Migrator](https://github.com/zendesk/zendesk_app_migrator), used and maintained as *built-in* template resource
+
+### Move new app scaffold to master branch
+
+### Add support in [ZAT](https://github.com/zendesk/zendesk_apps_tools) to create new app with the new scaffold
+* New command option `zat new scaffold` will create a v2 app using the new app scaffold.
```

Comment:
not `--scaffold`?

Reply:
I don't know what's the standard in `zat`'s interface, but I'd stick to whatever it is.
If `scaffold` is supposed to be a sub-command, and there are other sub-commands, then this should be fine. If everything past the first subcommand is a flag, then we should keep the tool's interface consistant.

Reply:
good catch, miss it. Should be option --scaffold